porick/lib.py

```diff
@@ -1,12 +1,48 @@
 import bcrypt
 import re
+import smtplib
+
+from email.mime.text import MIMEText
 
 from flask import request
 from sqlalchemy import or_
 
 from models import User
 from porick import db
-from settings import PASSWORD_SALT
+from settings import PASSWORD_SALT, SERVER_DOMAIN, PASSWORD_RESET_REQUEST_EXPIRY, SMTP_REPLYTO, SMTP_SERVER
+
+
+
+reset_password_text = """
```

Comment:
Just reached the bottom of the diff, and i see we have a `/pw_reset` templates dir, so I'd simply stick the template in there in that case.